sql/dql/basic-queries/where-3.md

```diff
@@ -30,38 +32,90 @@ aspects:
+| Venusaur   | 525   | 80 | 82     | 83      |
+| Charmander | 309   | 39 | 52     | 43      |
+| Charmeleon | 405   | 58 | 64     | 58      |
+| Charizard  | 534   | 78 | 84     | 78      |
+| Squirtle   | 314   | 44 | 48     | 65      |
+| Wartortle  | 405   | 59 | 63     | 80      |
+| Blastoise  | 530   | 79 | 83     | 100     |
+| Caterpie   | 195   | 45 | 30     | 35      |
+| Metapod    | 205   | 50 | 20     | 55      |
+
+The `IN` condition allows you to easily test if a value inside a column matches any one specific item in a list of values.
 
 ```sql
 SELECT *
 FROM pokemon
-WHERE name IN ('charmander',
-  'squirtle', 'bulbasaur');
+WHERE name IN ('Charmander',
+  'Squirtle', 'Bulbasaur');
 ```
 
-The n-tuple list of possible values given to `IN` is the list of possible values that the attribute must take in that column for it to be included in the query result. In this case, we will only get the rows of the three starter Pokémon: Charmander, Squirtle and Bulbasaur.
+In this case, we will only get the rows of the three starter Pokémon: Charmander, Squirtle and Bulbasaur.
 
-The `LIKE` statement accepts a pattern string, where wildcard characters are used to denote an acceptable pattern. *Note that some databases may have a different implementation of the specific wildcard characters*
+| name       | total | hp | attack | defense |
+|------------|-------|----|--------|---------|
+| Bulbasaur  | 318   | 45 | 49     | 49      |
+| Charmander | 309   | 39 | 52     | 43      |
+| Squirtle   | 314   | 44 | 48     | 65      |
+
+The `LIKE` statement accepts a pattern string, where wildcard characters are used to denote an acceptable pattern.[1]
 
  - `%` is a **wildcard**. This means it counts for a string of any characters of any length, including nothing.
  - `_` represents only one character.
 
-Therefore, `%chu` denotes any string that ends in 'chu'. `_chu` denotes any string with only one character before 'chu'. They can be chained together into complex patterns like `___taco%` which denotes any string containing 'taco' starting on the 4th character and with any number of characters after.
+Therefore, `%le` denotes any string that ends in 'le'. `_le` denotes any string with only one character before 'le'. They can be chained together into complex patterns like `___taco%` which denotes any string containing 'taco' starting on the 4th character and with any number of characters after.
+
+```sql
+SELECT *
+FROM pokemon
+WHERE name LIKE '%le';
+```
+
+The result of this statement is the two rows of the Pokémon whose names end in '-le': Squirtle and Wartortle.
+
+| name      | total | hp | attack | defense |
+|-----------|-------|----|--------|---------|
+| Squirtle  | 314   | 44 | 48     | 65      |
+| Wartortle | 405   | 59 | 63     | 80      |
+
+Instead of chaining two comparison operators in your `WHERE` clause you can use the `BETWEEN` operator to define a range of acceptable values, like so:
```

Comment:
> Instead of chaining two comparison operators in your `WHERE` clause

I don't think you should start with this as it's not obvious how BETWEEN is equivalent to that. maybe 

add a short paragraph as a footnote:

"This can be seen as a shortcut for chaining two comparison operators with the `AND` clause"
-> show the equivalent code

Reply:
added the following footnote

This can be seen as a shortcut for chaining two comparison operators with the `AND` clause:
```sql
SELECT * FROM table_name
WHERE col BETWEEN 1 AND 10;
-- is the same as
SELECT * FROM table_name
WHERE col >= 1 AND
      col <= 10;
```